README.md

```diff
@@ -35,251 +13,97 @@ After you're finished please remove all the comments and instructions!
+## Project Description .
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+This capstone project is a website built for a social tech venture called [DEV-OPS] that gives senior high school graduates in deprived communities who for what ever reason could not enter into University a new hope and dream of training them to become software developers and the opportunity to lift themselves from poverty.
 
-### Install
+## Built With
 
-Install this project with:
+- Html, Css ,JavaScript
 
-<!--
-Example command:
+## Live Demo
+["click here"](https://admirerbrown.github.io/capstone-one/)
 
-```sh
-  cd my-project
-  gem install
-```
---->
+## Video Walkthrough
+["click here"](https://www.loom.com/share/4a25ad640eba4deab20dc67883137388)
 
-### Usage
+## Getting Started
 
-To run the project, execute the following command:
+To get a local copy up and running follow these simple steps.
 
-<!--
-Example command:
+## Prerequisites
 
-```sh
-  rails server
-```
---->
+- A computer running any operating system.
+- an internet connection
+- Have Git installed
+- A modern browser eg. Google Chrome
+- Knowledge of basic git commands
 
-### Run tests
+## Setup
 
-To run tests, run the following command:
+- link your computer to github
+- clone the repository
+- open the file in your code editor
+- set up Html and css linters for the project
 
-<!--
-Example command:
+## Install
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- install node_models and project dependencies
+- ensure there are no errors
 
-### Deployment
+## Usage
 
-You can deploy this project using:
+- run the project to ensure there are no linter errors
+- open index.html in your browser of choice
+- Enjoy!
 
-<!--
-Example:
+## Author
 
-```sh
+Samuel N. Kyere
 
-```
- -->
+GitHub: @admirerbrown
+Twitter: @brown_admirer
+LinkedIn: Samuel Ntow kyere
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
+## Contributing
 
 Contributions, issues, and feature requests are welcome!
 
```

Comment:
- [ OPTIONAL ] It would be better if you include the link to the issues page so that someone can easily navigate to the issues page and contribute to your project. :joy: